tests/manager/manager-ctor-wireversion.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): wire version support
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new \MongoDB\Driver\Manager(URI);
+
+$command = new MongoDB\Driver\Command(['ping' => 1]);
+try {
+    $manager->executeCommand("test", $command);
+} catch (\MongoDB\Driver\Exception\RuntimeException $e) {
+    if ($e->getCode() == 15) { // Bad Wire Version
+        echo "Bad wire version detected: ", $e->getMessage(), "\n";
```

Comment:
[PHPC-1225](https://jira.mongodb.org/browse/PHPC-1225) purposefully suggests not skipping the test because we don't have a live server, which explains why the `SKIPIF` above is quite empty. If I understand correctly, since we only echo for a wire protocol error this test will still pass when no server is listening on `URI` (since the error code will differ). If so, LGTM.

Reply:
Correct:

```
derick@singlemalt:~/dev/php/derickr-mongo-php-driver $ echo $MONGODB_URI
derick@singlemalt:~/dev/php/derickr-mongo-php-driver $ make test TESTS=tests/manager/manager-ctor-wireversion.phpt | grep PASS
PASS MongoDB\Driver\Manager::__construct(): wire version support [tests/manager/manager-ctor-wireversion.phpt] 
```